docs/openapi/schemas/Credential.yml

```diff
@@ -1,6 +1,7 @@
 type: object
 required:
   - '@context'
+  - id
```

Comment:
@brownoxford , shouldn't this be on VerifiableCredential.yml (not Credential.yml)? 
Requiring this from the requestor breaks with both options 1 and 4 of https://github.com/w3c-ccg/traceability-interop/issues/468.

Reply:
Regarding #468, this ticket is simply to bring our spec in line with the normative language that exists today - it is wholly separate from the changes that may come about from #468.

As for whether the ID should be required in Credential or VerifiableCredential yaml files - Based on responses from @OR13 and @mkhraisha, I have put the requirement on the base Credential.ym since VerifiableCredential is described as being a credential with an added proof and that normative statements should apply to both.

I'm expecting that we will need to make adjustments once #468 is resolved.

Reply:
Makes sense, @brownoxford